### PR TITLE
new: add `Troubleshooting` Guides + Dedicated `Metrics` section

### PR DESCRIPTION
**Pull request by incertum**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

> /kind event

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area community

**What this PR does / why we need it**:

Add new help guides.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/docs/install-operate/help/start-up-error.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Help, Falco Is Not Starting Up!
+description: Follow this guide to identify whether an issue or misconfiguration is causing Falco to crash or preventing it from starting up
+weight: 10
+---
+
+## Action Items (TL;DR)
```
Comment (incertum) [inline content/en/docs/install-operate/help/start-up-error.md:7]: @Andreagit97 and @mikegcoleman is this complementing the planned revamp of the Install and Operate Guides. Would you have additional suggestions?

File: content/en/docs/install-operate/help/dropping.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
+---
+title: Help, Falco Is Dropping Syscalls Events!
```
Comment (incertum) [inline content/en/docs/install-operate/help/dropping.md:2]: @FedeDP 😅 we finally have it ...

File: content/en/docs/install-operate/help/missing-fields.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Furthermore, sometimes Linux may not operate exactly as expected. One concrete example is that shell built-ins like `echo` do not cause a new spawned process, and the `echo` command does not get logged with Falco. Similarly, if a base64 encoded string gets interpreted during decoding, you do not have the original base64 blob in the command args unless the command was passed with the `sh -c` flag. Lastly, some fields only work for certain kernel versions or system configs (e.g. [proc.is_exe_upper_layer](https://falco.org/docs/reference/rules/supported-fields/#field-class-process) requires a container overlayfs).
+
+## Missing Container Images
```
Comment (incertum) [inline content/en/docs/install-operate/help/missing-fields.md:21]: @leogr I really hope we make this clearer, I'll post in one of the open issues that perhaps we should reconsider adding `host` to the image if it's a host process and for Falco 0.38 even `pod_sandbox_container` once the relevant libs PR is merged ...

File: content/en/docs/install-operate/help/missing-fields.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+- To expedite lookups, attempt to disable asynchronous CRI API calls by using the `--disable-cri-async` command line option when running Falco.
+- If you run the upstream Falco rules containing the `%container.info` placeholder output field, run Falco with the `-pc` or `-pk` command line option to automatically include and resolve crucial container and Kubernetes-related output fields. For additional details, consult the [Outputs Formatting](../../../outputs/formatting/) Guide, and consider adding more [Supported Output Fields](../../../reference/rules/supported-fields/#field-class-container).
+- Falco monitors both host and container processes. If the `container.id` is set to `host`, it indicates that the process is running on the host, and therefore, no container image is associated with it.
+
+{{% pageinfo color=info %}}
+The `k8s.*` fields are extracted from the container runtime socket simultaneously as we look up the `container.*` fields from the CRI API calls responses.
+{{% /pageinfo %}}
+
+Carefully read the field description documentation:
```
Comment (incertum) [inline content/en/docs/install-operate/help/missing-fields.md:35]: We updated them in libs for improved technical clarity already a while ago.

File: content/en/docs/install-operate/production-performance.md [hunk 5]
```diff
@@ -40,27 +36,146 @@ Top metrics:
...
+  state_counters_enabled: true
+  kernel_event_counters_enabled: true
+  libbpf_stats_enabled: true
+  convert_memory_to_mb: true
+  include_empty_values: true
+```
+
+Here is a brief glossary of the currently supported metrics. The snippet was retrieved from a more or less idle test `x86_64` Linux machine. Therefore, counters and event rates are very low, and note that `aarch64` will have slightly different kernel tracepoints.
+
+```yaml
```
Comment (incertum) [inline content/en/docs/install-operate/production-performance.md:78]: @Andreagit97 😅 finally ... it's up!

**Comment by leogr** [comment]
cc @LucaGuerra @mikegcoleman

File: content/en/docs/install-operate/help/dropping.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+
+Test 4: network connect
+
+```yaml
+base_syscalls:
+  custom_set: [clone, clone3, fork, vfork, execve, execveat, getsockopt, socket, connect, close]
+  repair: false
+```
+
+Test 5: open* syscalls
```
Comment (Issif) [inline content/en/docs/install-operate/help/dropping.md:166]: why some `*`?

File: content/en/docs/install-operate/help/start-up-error.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+- Check if all preconditions to start up the kernel drivers are met. Common issues include: 
+   - For each `ebpf*` driver, the `bpf` syscall needs to be allowed and not blocked by SELinux or similar. 
+   - Ensure the DKMS package is installed for the `kmod` driver, and your system may require custom-signed kernel modules. Also, verify the availability of the host `/dev` mount (e.g. `/dev:/host/dev` when running Falco over a container).
+   - In general, check that Falco has all host mounts when running from a container or as a daemonset in Kubernetes. Critical mounts for running Falco, assuming the kernel driver is available, include: `/etc:/host/etc`,  `/proc:/host/proc`, `/boot:/host/boot`, `/dev:/host/dev`.
+- For `ebpf` and `kmod` drivers, the kernel object code needs to be available for the exact kernel release (`uname -r`) of your system. This invites a wide range of possible issues:
+  - Assuming you use Falco's open-source artifacts and open-source kernels, one source of error can be that the pre-built kernel driver (published by The Falco Project) is not available for download.
```
Comment (Issif) [inline content/en/docs/install-operate/help/start-up-error.md:36]: User can explore this site to know which drivers are available https://download.falco.org/driver/site/index.html

Comment (incertum) [reply content/en/docs/install-operate/help/start-up-error.md]: @Issif added, btw nice suggestion!

**Comment by incertum** [comment]
Re-did the metrics fields explanations section, so that it is less crowded and more organized.

**Comment by incertum** [comment]
@leogr and @mikegcoleman changed my mind and create 2 new top level sections, namely "Metrics" and "Help!". Is it better?

**Comment by incertum** [comment]
As we are deciding to start deprecating the old drop stats / alerts we could consider removing https://falco.org/docs/event-sources/kernel/dropped-events/ once the metrics page is up. WDYT?

**Comment by incertum** [comment]
Also https://falco.org/docs/event-sources/kernel/dropped-events/#actions-rate-throttling is actually already removed CC @Andreagit97

**Comment by leogr** [comment]
> As we are deciding to start deprecating the old drop stats / alerts we could consider removing https://falco.org/docs/event-sources/kernel/dropped-events/ once the metrics page is up. WDYT?

Keep this on hold for now.

**Comment by mikegcoleman** [comment]
@incertum should this be /area docs instead of /area blog?

**Comment by mikegcoleman** [comment]
@incertum I think it'd be better if the top level topic was "troubleshooting" instead of "help" - when I see help I think of in-app help, which is basically what documentation is so it seemed a little odd.

**Comment by incertum** [comment]
> @incertum I think it'd be better if the top level topic was "troubleshooting" instead of "help" - when I see help I think of in-app help, which is basically what documentation is so it seemed a little odd.

Happy to change it if everyone is on board. On the other hand you often have help slack channels. Not sure what the best naming would be.

**Comment by leogr** [comment]
> > @incertum I think it'd be better if the top level topic was "troubleshooting" instead of "help" - when I see help I think of in-app help, which is basically what documentation is so it seemed a little odd.
> 
> Happy to change it if everyone is on board. On the other hand you often have help slack channels. Not sure what the best naming would be.

I usually assume the whole document is already a "help" guide. So "troubleshooting" sounds a bit better for me. No strong opinion anyway.

**Review by leogr — APPROVED** [review]

**Pull request merged**